common/djangoapps/microsite_configuration/__init__.py

```diff
@@ -1 +1,19 @@
+from django.conf import settings as base_settings
+
+from microsite_configuration import microsite
 from .templatetags.microsite import page_title_breadcrumbs
+
+
+class MicrositeAwareSettings():
+    """
+    This class is a handy utility to make a call to the settings
+    completely microsite aware by replacing the:
+    from django.conf import settings
+    with:
+    from microsite_configuration import settings
+    """
+
+    def __getattr__(self, name):
+        return microsite.get_value(name, base_settings.__getattr__(name))
+
+settings = MicrositeAwareSettings()
```

Comment:
We where recently bit by this. We created an override of all the templates using the capitalized form(devstack had the microsite config capitalized) and later on, it failed on stage. From what I recall of this day, there where about ~120 occurrences of this. I would support rectifying this now.